corehq/apps/app_manager/util.py

```diff
@@ -633,7 +633,7 @@ def get_form_source_download_url(xform):
 
     try:
         form = app.get_forms_by_xmlns(xform.xmlns)[0]
-    except KeyError:
```

Comment:
was `KeyError` intended to catch https://github.com/dimagi/commcare-hq/blob/4cdf4d17d2ca10724d55ae1a2455827f81d13909/corehq/apps/app_manager/models.py#L5341

Is that impossible to hit now?

Reply:
That was likely the intent, but it's always been impossible to hit, because `self.get_xmlns_map()` returns a `defaultdict`.